plugins/modules/purefa_pgsnap.py

```diff
@@ -917,7 +924,11 @@ def main():
     )
 
     required_if = [("state", "copy", ["suffix", "restore"])]
-    mutually_exclusive = [["now", "remote"]]
+    mutually_exclusive = [
+        ["now", "remote"],
+        ["overwrite", "add_to_pgs"],
+        ["overwrite", "with_default_protection"],
```

Comment:
Below condition check in Line [985](https://github.com/Pure-Storage-Ansible/FlashArray-Collection/pull/719/files#diff-878535a46b810c4e3849ed682a500ba0d124a008cb428c6b8b7f60cc3d36b00eR989) causing errors as "with_default_protection" default value set to true in Line [920](https://github.com/Pure-Storage-Ansible/FlashArray-Collection/pull/719/files#diff-878535a46b810c4e3849ed682a500ba0d124a008cb428c6b8b7f60cc3d36b00eR920). Removing default helped in testing, please check.

Reply:
Hmm. Given the line 985 check we don't need these exclusives for overwrite... We have to have a default value for `with_default_protection`.